### ISSUE TITLE
Support for bazel

### ISSUE THREAD

**Issue by jeffzoch** [Madrapps/dagger-plugin#44]
Hi, 
I am trying to use this plugin with a bazel project on intellij and nothing shows up. Just wondering if you had any plans to support bazel projects in intellij

**Comment by thsaravana** [comment]
No. There are no plans for supporting Bazel projects. I am adding the "help wanted" label and will keep this issue open for a while, to see if I get some help.
